### ISSUE TITLE
Couple of enhancements

### ISSUE THREAD

**Issue by krijnsent** [pietervdvn/MapComplete#238]
First of all: the site works pretty nice and helps me to add stuff. 2 thoughts:
-could the icons displaying e.g. an artwork also give some info on what information has already been filled? As a concept: for an artwork you could e.g. shrink the image a bit and add 4 properties/circles (e.g. "Has image", "Has artist", "Has art type", "Has wiki-data") like so: 
![image](https://user-images.githubusercontent.com/21982451/114014826-516cb980-9869-11eb-8018-7d0bdde3e308.png)
That would help me e.g. identifying missing information much easier.
-for artwork I can't add a name/title
-I was using the personal theme and accidentally added a bookcase where I wanted to add a mural. That is quite easy to do as the default type selected seems the type you used before. So could I a) always get a mandatory selection option and/or b) get the option to undo an accidental add? (Now deleted it through osm.)

**Comment by krijnsent** [comment]
Thanks for the extensive reply. I am getting the hang of Github, but adding stuff to other repositories is a whole new beast to me. So I'll gladly contribute, but will indeed try to start with some baby steps as you describe :-).

**Comment by pietervdvn** [comment]
The workflow to request changes in other repositories is explained [here](https://guides.github.com/activities/forking/)

TLDR:
1) make a copy of this repo (forking)
2) Make changes to your copy
3) Ask me to review your changes in the format of a pull request

**Comment by pietervdvn** [comment]
Closing as stale
